### PR TITLE
Use preview api instead of store 

### PR DESCRIPTION
**Pull request by kasperpeulen**
And fix related deprecation messages

Is this a patch release @yannbf?

<details>
  <summary>📦 Published PR as canary version: <code>0.9.5--canary.273.08bdc9c.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @storybook/test-runner@0.9.5--canary.273.08bdc9c.0
  # or 
  yarn add @storybook/test-runner@0.9.5--canary.273.08bdc9c.0
  ```
</details>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -110,7 +110,7 @@
     "@storybook/core-common": "next",
     "@storybook/csf": "next",
     "@storybook/csf-tools": "next",
-    "@storybook/store": "next",
+    "@storybook/preview-api": "next",
```
Comment (ndelangen) [inline package.json:113]: I'd expect this change to have some runtime impact?

Comment (kasperpeulen) [reply package.json:113]: fixed

**Comment by kasperpeulen** [comment]
Hm.. what are those security alerts

**Review by yannbf — APPROVED** [review]
LGTM!

**Pull request merged**